d2game/d2gamescreen/main_menu.go

```diff
@@ -234,6 +374,10 @@ func (v *MainMenu) loadBackgroundSprites() {
 	v.serverIPBackground.SetPosition(serverIPbackgroundX, serverIPbackgroundY)
 }
 
+func translateLabel(label int, lng string, asset *d2asset.AssetManager) string {
+	return asset.TranslateString(fmt.Sprintf("#%d", baseLabelNumbers(label+d2resource.GetLabelModifier(lng))))
+}
+
```

Comment:
I don't think that this logic belongs in the main menu. Retrieving the correct string for the current locale should be something that the asset manager deals with, not the main menu.